tests/test_params_repr.py

```diff
@@ -0,0 +1,46 @@
+import pytest
+from fastapi.params import Body, Cookie, Depends, Header, Param, Path, Query
+
+test_data = ["teststr", None, ..., 1, []]
+
+
+def get_user():
+    return {}  # pragma: no cover
+
+
+@pytest.fixture(scope="function", params=test_data)
```

Comment:
Very clever! I actually didn't know of these fixtures with `params` :nerd_face: :heavy_check_mark: